lib/src/util.dart

```diff
@@ -1,4 +1,72 @@
 import 'dart:convert';
 
+import 'package:charcode/charcode.dart';
+
 String escapeHtml(String html) =>
     const HtmlEscape(HtmlEscapeMode.ELEMENT).convert(html);
+
+// Escape the contents of [value], so that it may be used as an HTML attribute.
+
+// Based on http://spec.commonmark.org/0.28/#backslash-escapes.
+String escapeAttribute(String value) {
+  var result = new StringBuffer();
+  var codeUnits = value.codeUnits;
+  int ch;
+  for (var i = 0; i < codeUnits.length; i++) {
```

Comment:
I believe it's faster to iterate through a string using `value.codeUnitAt()` rather than indexing into `codeUnits`.

Reply:
Done